qutebrowser/browser/commands.py

```diff
@@ -281,6 +307,9 @@ def openurl(self, url=None, implicit=False,
                     else:
                         # Explicit count with a tab that doesn't exist.
                         return
+                elif curtab.pin is True:
+                    message.info("Tab is pinned!")
+
```

Comment:
Please remove the blank line here - it makes me think the if-block is over here, while an `else:` actually follows :wink: